app/views/hub/messages/index.html.erb

```diff
@@ -5,87 +5,87 @@
   <%= render "hub/clients/client_header" %>
 
   <%= render "hub/clients/navigation" %>
-  <div class="hub-section">
-    <h2>Contact History</h2>
-  </div>
-  <% if @messages_by_day.present? %>
-    <ul class="message-list" aria-live="polite" data-js="messages-pub-sub" data-client-id="<%=@client.id%>">
-
-      <% @messages_by_day.each_with_index do |(datetime, contact_records), day_index| %>
-        <li class="message__day-heading"><%= date_heading(datetime) %></li>
-
-        <% contact_records.each_with_index do |contact_record, contact_record_index| %>
-          <% if contact_record_index == (contact_records.size - 1) && day_index == (@messages_by_day.size - 1) %>
-            <%= render "shared/message_list_contact_record", contact_record: contact_record, is_last_item: true %>
-          <% else %>
-            <%= render "shared/message_list_contact_record", contact_record: contact_record %>
-          <% end %>
-        <% end %>
-
-      <% end %> <%# end of day %>
-
-    </ul>
-  <% else %>
-    <%#
-      In order to hide this message list using :empty css pseudo selector and
-      not break the realtime messaging functionality, we need to render an
-      element that has no body whatsoever (even whitespace).
-      https://css-tricks.com/almanac/selectors/e/empty/#will-not-match
-    %>
-    <ul class="message-list" data-js="messages-pub-sub" data-client-id="<%=@client.id%>"></ul>
-  <% end %>
-
-  <div class="slab" id="composer">
-    <div class="communication-preferences">
-      <% unless @client&.intake.email_address.present? || @client&.intake.sms_phone_number.present? %>
-        <%= t(".no_way_to_communicate") %>
-      <% else %>
-        <%= t(".preference_intro") %>
-        <ul class="with-bullets">
-          <% if @client&.intake.sms_notification_opt_in_yes? && @client&.intake.sms_phone_number.present? %>
-            <li><%= t("general.text_message") %></li>
+  <div class="slab slab--padded">
+    <div class="client-container">
```

Comment:
I noticed that this was used to set a fixed width. Alternatively, we could use the honeycrisp grid system here, with a grid item width.